### PR TITLE
Old webdav configuration is not supported anymore

### PR DESCRIPTION
**Pull request by nterray**
Bonus: indicates where platform administrators can find webdav activity log file.

### CODE REVIEW

File: languages/en/deployment-guide/13.x.rst [hunk 1]
```diff
@@ -8,6 +8,12 @@ Tuleap 13.7
 
   Tuleap 13.7 is currently under development.
 
+Webdav
+------
+
+The access via a dedicated host (like ``https://webdav.tuleap.example.com/``) that was once necessary for windows 7 and
```
Comment (LeSuisse) [inline languages/en/deployment-guide/13.x.rst:14]:
```suggestion
The access via a dedicated host (like ``https://webdav.tuleap.example.com/``) that was once necessary for Windows 7 and
```

Comment (LeSuisse) [inline languages/en/deployment-guide/13.x.rst:14]: I'm not sure to understand the sentence, the "and before support, is now removed" seems a bit weird to me. Can you rephrase it?

**Review by LeSuisse — CHANGES_REQUESTED** [review]

File: languages/en/deployment-guide/13.x.rst (hunk 1 above)
Comment (nterray) [reply languages/en/deployment-guide/13.x.rst]: Done

Comment (nterray) [reply languages/en/deployment-guide/13.x.rst]: Done

**Review by LeSuisse — APPROVED** [review]
